common/app/templates/inlineJS/nonBlocking/interactiveFonts.scala.js

```diff
@@ -0,0 +1,7 @@
+var fonts = [].slice.apply(window.parent.document.styleSheets)
```

Comment:
I'm going to leave this for now. The `resize` method especially could actually be useful in the embeds for which want to update their size